.github/workflows/cleanup-pr-app.yaml

```
@@ -39,18 +49,40 @@ jobs:
-              try(rsconnect::purgeApp(appName = nm,
-                account = Sys.getenv("SHINY_ACCOUNT"), server = "shinyapps.io"))
-              cat(nm, "terminated and purged\n")
-            }'
+            exists <- function() {
+              apps <- rsconnect::applications(account = acct,
+                                              server = "shinyapps.io")
+              nm %in% apps$name
+            }
+            if (!exists()) {
+              cat(nm, "does not exist - nothing to purge
+")
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)